conda_smithy/templates/circle.yml.tmpl

```diff
@@ -17,8 +19,16 @@ dependencies:
   override:
     - docker pull condaforge/linux-anvil
 
+{% endif -%}
+{% endblock -%}
 test:
   override:
+{%- block env_test %}
+{%- if matrix %}
     # Run, test and (if we have a BINSTAR_TOKEN) upload the distributions.
     - ./ci_support/run_docker_build.sh
-{# Keep the terminal newline. #}
+{% else %}
+    # In case the skip fails for some reason, do a fast finish.
```

Comment:
Also, to help clarify skipped further. Please see this [list of "builds"]( https://circleci.com/gh/conda-forge/anypytools-feedstock )*. Also note that CircleCI calls them skipped if they do not run. This differs from Travis and AppVeyor where the skipped builds don't appear at all. Though they are still skipping builds just the same.

\* - One of them unfortunately is a real builds that shouldn't be. The rest of the real builds are intentional builds for testing purposes.